apps/wallet-mock/src/test/telemetry.cy.ts

```diff
@@ -9,44 +9,46 @@ const appConfigMock = {
 }
 
 describe('Telemetry dialog elements', () => {
-  // 0001-WALL-003 if the app sends telemetry/analytics: must be prompted to opt into (or stay out of) analytics
+  // 0001-WALL-003 - if the app sends telemetry/analytics: must be prompted to opt into (or stay out of) analytics
   before(() => {
     cy.mock('service.GetAppConfig', appConfigMock)
     cy.visit('/')
   })
   it('should see telemetry dialog', () => {
-    cy.get('[role="dialog"]').should('be.visible')
+    cy.getByTestId('telemetry-option-dialog').should('be.visible')
   })
   it('should see telemetry dialog title', () => {
-    cy.get('[role="dialog"] h2')
+    cy.getByTestId('telemetry-option-dialog')
+      .find('h2')
       .should('be.visible')
       .and('have.text', 'Report bugs and crashes')
   })
 
   it('should see info text', () => {
-    cy.get('[data-testid="telemetry-option-form"] > p').should(
-      'have.text',
-      'Selecting yes will help developers improve the software'
-    )
+    cy.getByTestId('telemetry-option-form')
```

Comment:
if we are refactoring this to use the new helper can we do the service file as well please :pray:

Reply:
Not really sure what this comment is about :)

Reply:
I'm being an idiot, dw 🙃. My bad!